src/main/java/org/codehaus/mojo/versions/UpdateParentMojo.java

```diff
@@ -54,6 +54,14 @@
     @Parameter( property = "parentVersion", defaultValue = "null" )
     protected String parentVersion = null;
 
+    /**
+     * to update parent version by force when it is RELEASE or LATEST
+     *
+     * @since 2.8
+     */
+    @Parameter( property = "forceUpdate", defaultValue = "False" )
+    protected Boolean forceUpdate = null;
```

Comment:
maybe should be `boolean` rather than `Boolean` to avoid NPE.

Reply:
agreed

Reply:
Done, Please approve.

Reply:
@olamy   Please help